mlir/test/Dialect/Vector/vector-transfer-flatten.mlir

```diff
@@ -131,10 +131,42 @@ func.func @transfer_read_dims_mismatch_non_contiguous_non_zero_indices(
 
 // -----
 
+/// The leading dynamic shapes don't affect whether this example is flattenable
+/// or not as those dynamic shapes are not candidates for flattening anyway.
+
+func.func @transfer_read_leading_dynamic_dims(
+    %arg : memref<?x?x8x4xi8, strided<[?, 32, 4, 1], offset: ?>>,
+    %idx_1 : index,
```

Comment:
NIT: As part of unifying variable names, `transfer_read_dims_mismatch_non_contiguous_non_zero_indices`'s indices args start from 0 and do not have an `_`.  `idx0 : index`

Reply:
Thanks for pointing this out! In fact, ATM  there are
*  4 instances of `idx1`, and 
* 12 instances of `idx_1` 

in this file. Let me make sure that we are indeed consistent, but I'll use `idx_1` rather than `idx1` - the former is already more common :)

Great work noticing this 😅 🙏🏻